plugins/cost-insights/config.d.ts

```diff
@@ -21,6 +21,11 @@ export interface Config {
      */
     engineerCost: number;
 
+    /**
+     * @visibility frontend
+     */
+    engineerThreshold: number;
```

Comment:
Should have a question mark if it's optional :)

Reply:
nice catch! 🙈